web/src/beta/hooks/useAssetUploader/hooks.tsx

```diff
@@ -9,20 +9,23 @@ export default ({
   workspaceId,
   onAssetSelect,
   assetType,
+  fileFormat,
 }: {
   workspaceId?: string;
   onAssetSelect?: (inputValue?: string, name?: string) => void;
   assetType?: string;
+  fileFormat?: "CSV" | "GeoJSON" | "KML" | "CZML";
```

Comment:
Can you make an ACCEPTED_FILE_FORMAT constant somewhere and use that everywhere we need file format?